<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StudentUI</class>
 <widget class="QMainWindow" name="StudentUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>413</width>
    <height>202</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="viewGradesButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Žiūrėti balus</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exitButton">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>70</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Atsijungti</string>
    </property>
   </widget>
   <widget class="QPushButton" name="viewSubjectsButton">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>70</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Peržiūrėti dėstomus
 dalykus</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
